modules/hibernate-4.2/src/test/java/org/apache/ignite/cache/hibernate/HibernateL2CacheMultiJvmTest.java

```diff
@@ -229,6 +242,8 @@ public void testL2Cache() throws Exception {
         SessionFactory startHibernate(String igniteInstanceName) {
             log.info("Start hibernate on node: " + igniteInstanceName);
 
+            Logger.getRootLogger().setLevel(org.apache.log4j.Level.INFO);
```

Comment:
Please add a comment on why this is needed

Reply:
Added comment.